shims/common/src/main/scala/org/apache/gluten/GlutenConfig.scala

```diff
@@ -2170,4 +2172,11 @@ object GlutenConfig {
           "Otherwise, do nothing.")
       .booleanConf
       .createWithDefault(false)
+
+  val HDFS_VIEWFS_ENABLED =
+    buildStaticConf("spark.gluten.storage.hdfs.viewfs.enabled")
```

Comment:
Some of these words can be combined with no period mark, which may look better.
-> `spark.gluten.storage.hdfsViewfs.enabled`